.github/workflows/publish-docker-image.yml

```diff
@@ -67,6 +71,6 @@ jobs:
           # as on the command line `--build-arg`, one to a line:
           # docker_build_args: BUILD_ARG_ONE=value1\nBUILD_ARG_TWO=value2
           # DUE TO JSON LIMITATIONS, NEWLINES MUST BE EXPLICITLY ADDED AS \n
-          docker_build_args: 'SDK_VERSION=${{ inputs.sdk_version }}'
+          docker_build_args: SDK_VERSION=${{ github.ref_type == 'tag' && github.ref_name || inputs.sdk_version }}
```

Comment:
@emre-aydin 
> Building new images with each tag might be problemmatic if tags are created before publishing Python modules to PyPI.

You're right that we create tags before we publish wheels to PyPI. For more details see steps 3 and 9 in the [SDK release process](https://zapatacomputing.atlassian.net/wiki/spaces/ORQSRUN/pages/461045791/SDK+release+process).

> Let me know if this is the case so that I can remove those triggers from the build and find another way to tag the images.

I think we could still make it work by installing from git source with something like 

```
SDK_REQUIREMENT="orquestra-sdk[ray] @ git+https://github.com/zapatacomputing/orquestra-workflow-sdk.git@{{github.ref_name}}"
```

Unfortunately I don't know GitHub's syntax enough to suggest a full ternary operator + string interpolation expression.

Alternatively, we can trigger it separately when the wheels are available on PyPI (step 9 in the [release process](https://zapatacomputing.atlassian.net/wiki/spaces/ORQSRUN/pages/461045791/SDK+release+process)).

Reply:
I would prefer having things as flexible as possible in the beginning so that we can learn and adapt the processes as we go. Let me remove the tag trigger and manually trigger the build as part of the release process in the beginning.